tests/contract_testcases/substrate/structs/parse_structs_22.sol

```diff
@@ -0,0 +1,11 @@
+contract Test {
+	struct S {
+		int foo;
+		S[][2] s;
+	}
+
+	function test(int f, S[][2] ss) public returns (S) {
+		return S(f, ss);
+	}
+}
+
```

Comment:
Just for completeness, you can add a test for a struct that holds function pointers.

Reply:
Good idea, added it